runtime/bcverify/vrfyhelp.c

```diff
@@ -748,9 +764,9 @@ isClassCompatibleByName(J9BytecodeVerificationData *verifyData, UDATA sourceClas
 
 	*reasonCode = 0;
 
-	/* NULL is magically compatible */
-	if( sourceClass == BCV_BASE_TYPE_NULL ) 
-		return (IDATA) TRUE;
+	/* NULL is compatible with non Q type classes */
+	if(sourceClass == BCV_BASE_TYPE_NULL)
```

Comment:
Please put the `return` statement in `{ }` .

Reply:
addressed this and your other comments in a2d8527